src/config.ts

```diff
@@ -105,6 +105,17 @@ Object.keys(mustHave).forEach(key => {
   }
 })
 
+/**
+ * Use this if "0" should be respected as 0 and only use the default value for
+ * other falsy values, such as `undefined`, `null`, or an empty string.
+ *
+ * Because JS treats 0 as falsy, the following returns 42: `Number("0") || 42`
```

Comment:
lol, JS

Reply:
❗️ 

I see a bunch of other `Number(...)` uses below, but none that seem to run into this same problem today. Good catch!

Reply:
Yeah I figured we'll know the issue when we run into it again and then update appropriately.